duckduckgo/duckduckgo-privacy-extension
# Update links that should semantically be buttons

**Reviewer:** @jagtalon 

## Description:
Adds the `role=button` attribute to the "X trackers blocked", "__ Privacy Practices" links in the hamburger menu, as well as the "Add to whitelist" link in the options menu.

Fixes #259 
Fixes #256 

## Steps to test this PR:

1. Open the Voiceover utility
2. Navigate to the buttons and check that Voiceover recognizes them.

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
